### ISSUE TITLE
Some db updates

### ISSUE THREAD

**Issue by dziban303** [rikgale/VRSData#5]
# 2023-02-28

```sql
AE0940 / USMC-UC35     # miscoded as PAT-C560

ADFF0B / 95-3009 / Raytheon Aircraft Company / Raytheon Aircraft Company T-6B Texan II / serial PT-013 / United States Air Force / RCH-TEX2
AE1155 / CNV-TEX2   # this one may already be in there, sorry
AE115A / CNV-TEX2
```

# 2023-03-01
```prolog
AE0E55 / United States Army / serial 70-993 / PAT-H60   # This one is a bit of a mystery.
AE04AD / PAT-C560
ADFF84 / RCH-T38
AE61DF / 761578 / Northrop / Northrop F-5N Tiger II / serial L1053 / United States Navy / CNV-F5
```

**Comment by rikgale** [comment]
AE115A - make your mind up ;) Navy or Air Force - you list this one on both days :')  I had is listed as Navy without the CNV-TEX2

AE61DF / 761578 - did this one have a BANDIT c/s out of Key West today?

Thanks for these, all added - keep 'em coming. All into will wind up in the refreshed DB this weeked!

**Comment by dziban303** [comment]
Lol, yeah, sorry. AE115A is Navy, CNV-TEX2. Verified with images on jetphotos.

---

AE61DF used `AAAAAAA` as its callsign and stuck close to NOLA; adsbx doesn't have any history of this plane going back through late December.

![image](https://user-images.githubusercontent.com/7931765/222282326-b45a78f3-3e5c-4ec7-a80b-992955a2653d.png)

---

There was also someone flying around with a miscoded hex, 000001. I tried to get outside and identify when he flew by, but it was a few miles away and I didn't have binocs. not that it'd matter much for the database. Kinda looked twin-tailed, but as we have both Hornets and Eagles here I couldn't say which it was.

![image](https://user-images.githubusercontent.com/7931765/222285284-2d4a6f00-261f-4dec-b30a-5f67cb9dc2de.png)

---

As I was scrolling around looking for the above miscode, I saw this odd serial number on 06A30D / A7-BFZ. Not sure if that's intended or not.

![image](https://user-images.githubusercontent.com/7931765/222283619-42ad1d98-3538-4646-9111-bb96608948ea.png)
